test/refactoring/SyntacticRename/backticks-noncollapsible-separate-arglabel.swift

```diff
@@ -5,4 +5,4 @@ Foo()/*test:ref*/[`x`: 10]
 
 // RUN: %empty-directory(%t.ranges)
 // RUN: %refactor -find-rename-ranges -source-filename %s -pos="test" -is-function-like -old-name "subscript(x:)" >> %t.ranges/backticks-noncollapsible-separate-arglabel.swift.expected
```

Comment:
Is `old-name` just used as a verification?

Reply:
Yes, used for verification that you don’t eg. rename an operator to a non-operator

https://github.com/apple/swift/blob/23effda44da844b450ae30b45ef721fe2508ee21/lib/Refactoring/SyntacticRename.cpp#L158-L195